frontend/src/hooks/api/actions/useFeatureStrategyApi/useFeatureStrategyApi.ts

```diff
@@ -2,14 +2,27 @@ import type {
     IFeatureStrategyPayload,
     IFeatureStrategy,
     IFeatureStrategySortOrder,
+    IConstraint,
 } from 'interfaces/strategy';
 import useAPI from '../useApi/useApi';
+import { useRecentlyUsedConstraints } from 'component/feature/FeatureStrategy/FeatureStrategyConstraints/RecentlyUsedConstraints/useRecentlyUsedConstraints';
 
 const useFeatureStrategyApi = () => {
     const { makeRequest, createRequest, errors, loading } = useAPI({
         propagateErrors: true,
     });
 
+    const { addItem: addToRecentlyUsedConstraints } =
+        useRecentlyUsedConstraints();
+
+    const saveConstraintsToRecentlyUsed = (constraints: IConstraint[]) => {
+        if (Array.isArray(constraints)) {
+            constraints.forEach((constraint) => {
+                addToRecentlyUsedConstraints(constraint);
+            });
+        }
```

Comment:
Small suggestion: would it be a better experience to have the addItem function instead be addItems, and you just add whatever. Then that function will take care of looping and all of that? Also encapsulates that logic a bit, I guess.